wscript

```diff
@@ -520,6 +520,7 @@ def configure(conf):
         conf.env.append_value('DEFINES', ['BT_CONTROLLER_CC2564X'])
     elif conf.is_asterix():
         conf.env.bt_controller = 'nrf52'
+        conf.env.imu_sensor = 'lsm6dso'
```

Comment:
my recommendation: I think that the `env.bt_controller` thing is kind of an anti-pattern that should (and will, now that we are all NimBLE!) die.  so I think things like this probably should move to `platform_capabilities` for now -- for instance, `asterix` and `obelix` probably should have `CAPABILITY_HAS_LSM6DSO` or something

Reply:
Happy to follow any configuration pattern. If you make this change in main, I'll rebase my branch to follow the same pattern.

Reply:
The build system is in general a big mess; we haven't spent time cleaning it up, and as of now it's growing organically from where it was left off ~10 years ago. I think though, that the platform capabilities way is wrong, as from what I get, it signals availability of interfaces, not any particular HW.